src/pybind/CMakeLists.txt

```diff
@@ -18,6 +18,30 @@ foreach(
   list(APPEND NMODL_PYTHON_FILES_OUT ${PROJECT_BINARY_DIR}/nmodl/${file})
 endforeach()
 
+add_library(pyembed ${CMAKE_CURRENT_SOURCE_DIR}/pyembed.cpp)
+set_property(TARGET pyembed PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+if(NOT LINK_AGAINST_PYTHON)
+  add_library(pywrapper SHARED ${CMAKE_CURRENT_SOURCE_DIR}/wrapper.cpp)
+else()
+  add_library(pywrapper ${CMAKE_CURRENT_SOURCE_DIR}/wrapper.cpp)
+  set_property(TARGET pywrapper PROPERTY POSITION_INDEPENDENT_CODE ON)
+  target_compile_definitions(pyembed PRIVATE STATIC_PYWRAPPER=1)
```

Comment:
I am not sure if Tristan document it in hpc coding convention but general suggestion was to name macros with project name i.e. `NMODL_STATIC_PYWRAPPER`. This way, they won't interfere if parent project accidentally introduce the same macros (e.g. in case of submodule).

Reply:
I forgot! fixed.